functions/src/export-csv.ts

```diff
@@ -237,38 +237,37 @@ function getValue(
   data: Pb.ITaskData[]
 ): string | number | null {
   const result = data.find(d => d.taskId === task.id);
-  if (!result) {
-    return null;
+  if (!result || result.skipped) return null;
+  const {
+    textResponse,
+    numberResponse,
+    dateTimeResponse,
+    multipleChoiceResponses,
+    drawGeometryResult,
+    captureLocationResult,
+    takePhotoResult,
+  } = result;
+  if (textResponse) return textResponse.text ?? null;
```

Comment:
The `if .. else if` idiom is safer in this case since it represents an exhaustive list of mutually exclusive possibilities.